carapace-server/src/main/resources/bin/service

```diff
@@ -113,7 +116,7 @@ start() {
     fi
   fi
 
-  $PRERUN "$BASE_DIR/bin/java-utils.sh -daemon $SERVICE -Dpidfile=$PIDFILENAME $MAINCLASS $OTHERPARAMS"
+  $PRERUN "$BASE_DIR/bin/java-utils.sh -daemon $SERVICE -Dpidfile=$PIDFILENAME $MAINCLASS $OTHERPARAMS $DB_CONFIG_FILE"
```

Comment:
It is better to let the service pick the default file server.properties
and not override it here.

with your change you can't run
`bin/service server start conf/mydemoconfig.properties`

this is very common across several projects